tests/integrated/test-dataiterator/runtest

```diff
@@ -6,59 +6,34 @@
 
 from __future__ import print_function
 try:
-  from builtins import str
+    from builtins import str
 except:
-  pass
+    pass
 from boututils.run_wrapper import shell, shell_safe, launch_safe, getmpirun
 from boutdata.collect import collect
 from sys import stdout, exit
 
 MPIRUN=getmpirun()
 
 print("Making DataIterator test")
-s, out = launch_safe("make > make.log",pipe=True)
-if s:
-  print("Failed to run make")
-  exit(s);
+s, out = shell_safe("make > make.log",pipe=True)
 
 flags = [""]
-#, "nsys=2", "nsys=5 periodic", "nsys=7 n=10"]
 
-code = 0 # Return code
 for nproc in [1,2,4]:
-  for mproc in [1,2,4,8]:
-    cmd = "./test_dataiterator"
+    for mproc in [1,2,4,8]:
+        cmd = "./test_dataiterator"
 
-    print("   %d processors...." % (nproc))
-    r = 0
-    for f in flags:
-        stdout.write("\tflags '"+f+"' ... ")
+        print("   %d processors...." % (nproc))
 
-        shell("rm data/BOUT.dmp.* 2> err.log")
+        for f in flags:
```

Comment:
Seeing as `flags` just has one element, the empty string, we don't need to iterate over it at all